server/routes/product.js

```diff
@@ -1,5 +1,8 @@
-import app from '../express'
+var express = require('express');
 import Product from '../controllers/product'
 
+var router = express.Router();
 
+router.get('/', Product.list);
 
+module.exports = router;
```

Comment:
hay que usar la misma syntax de export que en el resto 
```suggestion
export default router;
```